assets/js/blocks/classic-template/index.tsx

```diff
@@ -10,31 +16,43 @@ import { useBlockProps } from '@wordpress/block-editor';
 import { Button, Placeholder } from '@wordpress/components';
 import { __, sprintf } from '@wordpress/i18n';
 import { box, Icon } from '@wordpress/icons';
-import { useDispatch } from '@wordpress/data';
+import { select, useDispatch, subscribe } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
 import './style.scss';
-import { TEMPLATES } from './constants';
+import { BLOCK_SLUG, TEMPLATES } from './constants';
 
 interface Props {
 	attributes: {
 		template: string;
 		align: string;
 	};
+	setAttributes: ( attributes: Record< string, string > ) => void;
 	clientId: string;
 }
 
-const Edit = ( { clientId, attributes }: Props ) => {
+const Edit = ( { clientId, attributes, setAttributes }: Props ) => {
```

Comment:
This is optional but we can use the `BlockEditProp` type here, for example:

https://github.com/woocommerce/woocommerce-blocks/blob/f2ece167c36b56a8ec26f15fe573dbc22983a654/assets/js/blocks/price-filter/edit.tsx#L31-L34